VideoLocker/res/drawable/edx_input_background.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<shape xmlns:android="http://schemas.android.com/apk/res/android"
+    android:padding="10dp"
```

Comment:
Does padding defined as a top level attribute of `shape` actually work? In any case, we might want to use one of the predefined dimensions for it if we need it here (`edX.EditText` already seems to have it's padding defined, but `edX. SpinnerContainer` doesn't).